### PR TITLE
snappy: bugfix for snap.yaml parsing to be more consistent with the spec

### PR DESCRIPTION
**Pull request by mvo5**
This branch fixes the inconsistency of the snap.yaml parsing code in snappy with the spec.

### CODE REVIEW

File: docs/meta.md [hunk 1]
```diff
@@ -36,14 +36,15 @@ The following keys are optional:
...
-    * `stop`: (optional) the command to stop the service
+    * `stop-command`: (optional) the command to stop the service
     * `stop-timeout`: (optional) the time in seconds to wait for the
                       service to stop
     * `restart-condition`: (optional) if specified, use the given restart
       condition. Can be one of `on-failure` (default), `never`, `on-success`,
       `on-abnormal`, `on-abort`, and `always`. See `systemd.service(5)`
       (search for `Restart=`) for details.
-    * `poststop`: (optional) a command that runs after the service has stopped
+    * `poststop-command`: (optional) a command that runs after the service
```
Comment (niemeyer) [inline docs/meta.md]: post-stop-command, I think?  These are two words, right?

Comment (niemeyer) [inline docs/meta.md]: Ah, yeah, I see it's just a typo per implementation below

**Comment by niemeyer** [comment]
LGTM

File: docs/meta.md (hunk 1 above)
Comment (mvo5) [inline docs/meta.md]: Thanks, good catch! Fixed in git now.

**Pull request merged**